src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IDisposableExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    internal static class IDisposableExtensions
+    {
+        /// <summary>
+        /// Calls the <see cref="IDisposable.Dispose"/> method on an object, allowing the object to be null.
+        /// </summary>
+        internal static void DisposeIfNotNull(this IDisposable value)
```

Comment:
probably was written before 4.6, changing now

---
In reply to: [71806771](https://github.com/dotnet/roslyn-project-system/pull/309#discussion_r71806771) [](ancestors = 71806771)